### ISSUE TITLE
SolO IRFU quicklooks: Matrix dimensions must agree

### ISSUE THREAD

**Issue by ErikPGJ** [irfu/irfu-matlab#103]
### Step 1: Latest code?
- [X] This issue is present in the most recent code (i.e. after running ```git pull``` this is still a problem). Used commit 50761f28.

### Step 2: Describe your environment
 * irfu-matlab branch (e.g. ```master```, ```devel```): devel
 * Matlab version used (e.g. ```Matlab R2020b```): MATLAB R2022b
 * Operating system (e.g. ```Windows 10```, ```Mac OS X 10.15.6```, ```Linux Ubuntu 20.04```): Ubuntu 18.04.6 LTS
 - [X] A 64-bit installation of Matlab was used.

### Step 3: Describe the problem

Crashes when making the call `solo.quicklook_main('/home/erjo/so_irfu-matlab_qli//mission/solar_orbiter/+solo/irf_logo.png', '/data/solo/data_yuri/', '/home/erjo/temp/so_qli/so_qli_MTEST.2022-08-30_16.16.49.0_1_2020-02-12T00:00:00---2020-02-19T00:00:00', '0', '1', '2020-02-12T00:00:00', '2020-02-19T00:00:00');`.

```
Operating system is OK
Matlab version is OK
CDF_LEAPSECONDSTABLE is OK
saving caching status
DB caching enabled
  [warning: solo_db.get_variable/append_sci_var(131)] Discarded 1 data points
  [warning: solo_db.get_variable/append_sci_var(131)] Discarded 1 data points
exception.message=Matrix dimensions must agree.
pcolor, row 61
irf_spectrogram, row 270
quicklooks_7days, row 138
quicklook_main, row 317
```

Note: Not the same as #99 which concerned `quicklooks_24_6_2_h`. Copy-paste error?

**Comment by ErikPGJ** [comment]
Maybe fixed with a12ed124.

**Comment by ErikPGJ** [comment]
Solved with 844567b4 (or earlier)?

**Comment by ErikPGJ** [comment]
Test run using 555610 (commit is not the bugfix) implies that bug has been fixed.
